source/draw/Color.ooc

```diff
@@ -73,6 +77,8 @@ ColorYuva: cover {
 	toMonochrome: func -> ColorMonochrome { ColorMonochrome new(this y) }
 	toUv: func -> ColorUv { this toYuv() toUv() }
 	toYuv: func -> ColorYuv { ColorYuv new(this y, this u, this v) }
+	toRgb: func -> ColorBgr { this toYuv() toRgb() }
```

Comment:
Updated. My compiler did not catch that but something about "CharBuffer lastIndexOf()" is instantly crashing every time.